composer.json

```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^5.6 || ^7.0",
```

Comment:
this looks like a regression to me

Reply:
it is not

Reply:
ok, learned that its on purpose. but there is no point in allowing those as new phpcr-odm versions only support php 7.1 as well.

Reply:
If that's true, the 5.6 build should fail

Reply:
So did you drop versions of php-odm that allow 5.6 or not?

Reply:
hm, i guess the sonata bundle still supports phpcr-odm 1.x so it does make sense to keep this in, at least for now.